apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -16,6 +24,11 @@ defmodule AdminAppWeb.ProductView do
   @sort_field_keys ["rummage", "sort", "field"]
   @sort_order_keys ["rummage", "sort", "order"]
 
+  def active_stock_locaton() do
```

Comment:
@SagarKarwande I am not sure about this method either.

Reply:
We write some function in `view` file that are specifically needed for templates.

`active_stock_locaton/0` is used to show select dropdown for active Stock Locations. Phoenix forms `select` need the data in one of the following forms:
```elixir
[[key: "Pune", value: 1 ], [key: "Mumbai", value: 2]] OR
["Pune": 1, "Mumbai": 2] OR
[{"Pune", 1}, {"Mumbai", 2}]
```
This is something that is specific that is needed by Phoenix. So I have used the `StockLocation` model to fetch the data and later transform it in to the way the `select` input needs it.

[Select input doc](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Form.html#select/4)
> I'll change the naming of variables in the method.